src/lib.rs

```diff
@@ -51,7 +51,7 @@ pub struct CloudformationParseTree {
     pub resources: ResourcesParseTree,
     pub outputs: OutputsParseTree,
 
-    logical_lookup: HashSet<String>,
+    pub logical_lookup: HashSet<String>,
```

Comment:
Is this becoming `pub` just for testing? In that case, maybe `pub(crate)` is a less radical solution (generally speaking, I prefer to keep struct fields hidden unless the struct is "pure data", as otherwise your invariants could be broken by whoever owns the value)

Reply:
changed it to pub super, pub crate is too private